app/actions/definitions/documents.tsx

```diff
@@ -352,7 +352,10 @@ export const duplicateDocument = createAction({
  * of the collection for all collection members to see.
  */
 export const pinDocumentToCollection = createAction({
-  name: ({ t }) => t("Pin to collection"),
+  name: ({ t, stores }) => {
+    const activeCollectionName = stores.collections.active?.name;
+    return t("Pin to ") + activeCollectionName;
```

Comment:
This should use a template within the translation rather than string concatenation – not all languages are int the same order.

Reply:
Thanks for this info. I've pushed the changes...